csrc/multi_tensor_adam.cu

```diff
@@ -189,11 +189,10 @@ struct AdamCapturableFunctor
         int i = i_start + threadIdx.x + ii*blockDim.x;
         if(i < n && i < chunk_size)
         {
-	  g[i] = g[i] * (*inv_scale);
```

Comment:
If `g[i]` is in FP16 and the FP32 multiplication result is out of FP16 range, it may cause issues when the result is cast to FP16 to be stored in `g[i]`.